api/WIP/authentication.yaml

```diff
@@ -159,6 +280,291 @@ components:
           description: "JWT assertion token for the authenticated user"
           example: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..."
 
+    WebAuthnRegisterStartRequest:
+      type: object
+      properties:
+        user_id:
+          type: string
+          description: User ID of the user to register credentials for
+          example: "550e8400-e29b-41d4-a716-446655440000"
+        relying_party_id:
+          type: string
+          description: Relying Party ID (typically the domain)
+          example: "example.com"
+        relying_party_name:
+          type: string
+          description: Human-readable name for the Relying Party
+          example: "Example Corp"
+        authenticator_selection:
+          type: object
+          description: Criteria for selecting authenticators
+          properties:
+            authenticator_attachment:
+              type: string
+              description: Authenticator attachment modality
+              enum: ["platform", "cross-platform"]
+              example: "platform"
+            require_resident_key:
+              type: boolean
+              description: Whether a resident key is required
+              example: false
+            resident_key:
+              type: string
+              description: Resident key requirement
+              enum: ["discouraged", "preferred", "required"]
+              example: "preferred"
+            user_verification:
+              type: string
+              description: User verification requirement
+              enum: ["required", "preferred", "discouraged"]
+              example: "preferred"
+        attestation:
+          type: string
+          description: Attestation conveyance preference
+          enum: ["none", "indirect", "direct", "enterprise"]
+          example: "none"
+      required:
+        - user_id
+        - relying_party_id
+
+    WebAuthnRegisterStartResponse:
+      type: object
+      properties:
+        publicKeyCredentialCreationOptions:
```

Comment:
We're using both camel case and snake case in the payloads. Is it due to how these fields are returned from the browser APIs? If not, better to stick to a single format.

Applicable to other places as well

Reply:
Okay. Then shall we use camel case for all other properties such as user_id , relying_party_name, etc then?